content_scripts/courseReverseOrder.js

```diff
@@ -1,10 +1,7 @@
 let DEGREES = 0;
 const DEGREES_TO_REVERSE = 180;
 
-//TODO: Move CSS to CDN.
-const addButtonStyle = () => {
-    const style = document.createElement('style');
-    style.textContent = `
+const BUTTON_STYLE = `
```

Comment:
Maybe move it to a new file named `dashboard.css` and create a CDN for it, so the code will be neater?

Reply:
We already have file like that that Tamir did.  And this way we comply to Tamir's API
Regarding the changes you suggested seems fine for me. I'll commit it

Reply:
Update:  it's seems it's not possible to commit it through Github so if you can change it and push it will be great. 
Or if not I'll do in the evening. 
Just keep me update if you decide to do it

Reply:
Done.